rust_src/src/multibyte.rs

```diff
@@ -104,6 +104,11 @@ impl LispStringRef {
     pub fn as_mut_slice(&self) -> &mut [u8] {
         unsafe { slice::from_raw_parts_mut(self.data as *mut u8, self.len_bytes() as usize) }
     }
+
+    #[inline]
+    pub fn char_at(&self, index: ptrdiff_t) -> c_char {
```

Comment:
I'd propose calling this `byte_at` and returning `u8`.

Reply:
Thanks, done